buildSrc/src/main/kotlin/PluginDependencies.kt

```diff
@@ -0,0 +1 @@
+../../../../plugin/buildSrc/src/main/kotlin/PluginDependencies.kt
```

Comment:
Is this a symlink or something??

Reply:
yes, symlink. Without it `samples` project doesn't see `PluginVersions` class.